umap/admin.py

```diff
@@ -1,8 +1,31 @@
+import csv
+
+from django.contrib.auth.admin import UserAdmin as UserAdminBase
+from django.contrib.auth.models import User
 from django.contrib.gis import admin
+from django.http import HttpResponse
+from django.utils.translation import gettext_lazy as _
 
 from .models import DataLayer, Licence, Map, Pictogram, Team, TileLayer
 
 
+class CSVExportMixin:
+    actions = ["as_csv"]
+
+    @admin.action(description=_("CSV Export"))
+    def as_csv(self, request, queryset):
+        response = HttpResponse(
+            content_type="text/csv",
+            headers={"Content-Disposition": 'attachment; filename="export.csv"'},
```

Comment:
I would but a `csv_filename` on the class to get a comprehensible one (my download folder is a mess 🙈), like `umap_export_users.csv`? Even better with the date of the export?

Reply:
Or could I use the model name in lower case ?